ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -60,19 +60,16 @@ Item {
 
     RowLayout {
         id: searchInputWrapper
-        width: parent.width
+        width: 288
```

Comment:
Hardcoded sizes are bad practice :police_officer:

Reply:
@osmaczko I know! I would like to introduce a more 'sophisticated' solution which will apply to all - I've also complaint about this in discord :D